Automattic/pocket-casts-ios
# Reimagine Sharing: What's New

| 📘 Part of: #1910 |
|:---:|

Adds the "What's New" announcement for Reimagine Sharing.


https://github.com/user-attachments/assets/2774f060-dd4a-435a-8a13-b467cbe7fe08

https://github.com/user-attachments/assets/620b041d-0262-46bf-b150-69cd99439adb

https://github.com/user-attachments/assets/80ce36db-2804-4054-b3cf-7765b4963329

## To test

[7.71-Feature-Flag-Check.patch](https://github.com/user-attachments/files/16830725/7.71-Feature-Flag-Check.patch)

* Use the patch above or change the app version in `Info.plist` to 7.72
* Add this line to `AppDelegate.didFinishLaunching` to show the screen each time:
```swift
Settings.lastWhatsNewShown = "7.71"
```
* Ensure that the What's New screen is shown

## Checklist
